Configuration/Settings.yaml

```diff
@@ -8,3 +8,9 @@ Flowpack:
           host: 127.0.0.1
           port: 6379
           database: 15
+        # Set password in your redis config
+        passwordProtectedClient:
+          host: 127.0.0.1
+          port: 6379
+          password: 'My_Secret_Password'
+          database: 15
```

Comment:
If I get it right, this is only needed for the tests below, so it should be moved to `Configuration/Testing/Settings.yaml`

Reply:
You are right. But the other testing settings were also in the Settings.yaml.

Reply:
Ah, right. It's under `testing` like in the other queues. All good!